src/main/java/uk/gov/hmcts/reform/cpo/ApplicationParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.cpo;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Named
+@Singleton
+@Getter
+public class ApplicationParams {
+
+    @Value("${ccd.defaultPageSize}")
+    private String defaultPageSize;
```

Comment:
Think we should just be able to set this to `int` and then not worry about converting the `String` elsewhere this is used?

Reply:
DONE